pages/apis/rest_api/pipeline_templates.md

```diff
@@ -0,0 +1,293 @@
+# Pipeline templates API
+
+> 📘 Enterprise feature
+> Pipeline templates are only available on an [Enterprise](https://buildkite.com/pricing) plan.
+
+The pipeline templates API allows admins to create and manage pipeline templates for an organization.
+Non-admins will only be permitted to read pipeline templates marked as available by organization admins.
+
```

Comment:
I think this should link to the pipeline templates documentation so that a user can learn what they are actually used for.

Reply:
I like that suggestion, updated this in [39d7dd7](https://github.com/buildkite/docs/pull/2479/commits/39d7dd730b1b5c0910b26e9120bd3c3432ebbb7b)